src/main/rx/flysky.c

```diff
@@ -442,6 +452,21 @@ rx_spi_received_e flySkyDataReceived (uint8_t *payload)
 
     if (bound) {
         checkTimeout();
+#ifdef USE_RX_FLYSKY_SPI_LED
```

Comment:
I don't think in a case like this, where the hardware either has an LED or not, it makes much sense to use `USE_RX_FLYSKY_SPI_LED` and `RX_FLYSKY_SPI_LED_PIN`, since there is little use for a target in defining `USE_RX_FLYSKY_SPI_LED` without `RX_FLYSKY_SPI_LED_PIN`, so it should be only `RX_FLYSKY_SPI_LED_PIN`,

Opinions, @martinbudden, @jflyper?

Reply:
I would go for both USE_ and _PIN. (We just did the same with BEEPER...)

Reply:
Fair enough, disregard my comment then, @githubDLG.

Reply:
:smile: